packages/fiori/test/samples/Bar.sample.html

```diff
@@ -0,0 +1,135 @@
+<header>
+	<div class="control-header">Bar</div>
+	<div class="component-heading-since">
+		<span><!--since_tag_marker--></span>
+	</div>
+</header>
+
+<div class="component-package">@ui5/webcomponents</div>
```

Comment:
For the sample I would personally want to see more realistic examples like.
From the overview and from the markup the users will understand that they can place content at the sides and in the middle. 
And although really minor - perhaps you can order the sample either Footer, FloatingFooter, Header, Subheader or  Header, Subheader Footer, FloatingFooter

Hader
<img width="506" alt="Screenshot 2020-12-04 at 8 27 00 AM" 
src="https://user-images.githubusercontent.com/15702139/101129599-94442980-360a-11eb-90b7-2dbea5260c3a.png">

Footer
<img width="640" alt="Screenshot 2020-12-04 at 8 26 19 AM" src="https://user-images.githubusercontent.com/15702139/101129616-9ad2a100-360a-11eb-929a-cd99fda2a4f4.png">

Floating
<img width="504" alt="Screenshot 2020-12-04 at 8 28 17 AM" src="https://user-images.githubusercontent.com/15702139/101129675-b63dac00-360a-11eb-9d95-d7debe7cc0c9.png">